orchagent/dash/dashorchbase.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include <orch.h>
+#include "zmqserver.h"
+
+class ZmqConsumer : public ConsumerBase {
+public:
+    ZmqConsumer(swss::ZmqConsumerStateTable *select, Orch *orch, const std::string &name)
+        : ConsumerBase(select, orch, name)
+    {
+    }
+
+    swss::TableBase *getConsumerTable() const override
+    {
+        // ZmqConsumerStateTable is a subclass of TableBase
+        return static_cast<swss::ZmqConsumerStateTable *>(getSelectable());
+    }
+
+    void execute() override;
+    void drain() override;
+};
+
+class DashOrchBase : public Orch
```

Comment:
>DashOrchBase [](http://example.com/codeflow?start=6&length=12)

Can we move renaming into another separate PR? Seems like Dash is not an accurate name, because it is not bound  to dash project. Waiting community to provide more feedback on the name.

Other changes are easy to review and merge.

Reply:
Fixed, rename back to ZmqOrch